src/components/overlay-panel/overlay-panel.controller.ts

```diff
@@ -41,19 +41,11 @@ export class OverlayPanelController implements angular.IController {
 
   private showDialog = () => {
     this.ctrl.shown = true;
-
-    this.$timeout(() => {
-      this.ctrl.showOverlayPanel = true;
```

Comment:
It's not clear to me where this value was used, but looks like the only other reference to it is in the constructor. @rhamilto Can you remove that?